### ISSUE TITLE
CMS: For/ID attribute mismatch on Lat & long fields in Create Community Access point node

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#11108]
## Description
In the CMS, when an editor is creating a new Community Access Point node, there are 2 fields - one for Latitude and one for Longitude.  These fields have a mismatch for their labels, meaning the `id` of the `<input>` does not match the `for` attribute of the `<label>`.

For example, on the Latitude field, the `for` attribute is currently `edit-field-geolocation-0-value-lat` and the id of the <input> is `lat-edit-field-geolocation-0-value`.  

After looking at all the other Create nodes with Lat/Long fields as well as the edit nodes, I believe the `for=edit-field-geolocation-0-value-lat` is correct and that the id should be updated to match and be consistent.  Also, the longitude field should be updated accordingly as well.

URL of page with error: https://staging.cms.va.gov/node/add/vet_center_cap

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/194153505-a93b71d9-16de-4c40-a43f-e5fb30cbb552.png)

### Additional Context
I noticed that this seemed to be the only place these fields were incorrect.  I know Facilities had done work on these - I'm curious if there is a reason these do not match the mark up of the other Lat/long fields?

## Acceptance Criteria
- [ ] Update the Latitude field so the for/id matches
- [ ] Update the Longitude field so that the for/id matches

**Comment by swirtSJW** [comment]
I am not sure why these do not match the other uses, other than this is the only place where this field would be editable by users, the other instances are all locked down because they come in via upstream data source.

**Comment by omahane** [comment]
@swirtSJW @laflannery It turns out that we use different widgets in the Manage Form Display, depending on whether we show the map or not. In the Vet Center CAPs, we use the GeofieldMap Widget, rather than the Latitude/Longitude widget.
Lines 156 - 157 of `/geofield_map/src/Element/GeofieldMap.php` are as follows:
```
    $element['lat']['#attributes']['id'] = 'lat-' . $element['#id'];
    $element['lon']['#attributes']['id'] = 'lon-' . $element['#id'];
```
They should be
```
   $element['lat']['#attributes']['id'] = $element['#id'] . '-lat';
   $element['lon']['#attributes']['id'] = $element['#id'] . '-lon';
```
After making that change locally, the output appears correct (see attached).
![Screen Shot 2022-10-27 at 9 45 17 AM](https://user-images.githubusercontent.com/766573/198323269-96f04107-27ab-4747-b335-ff5c06e62310.png)

**Comment by omahane** [comment]
@swirtSJW I looked at the module and didn't find any issues related to this, so I created this one:
https://www.drupal.org/project/geofield_map/issues/3317875

I think Laura should get some credit for noticing this, but I'm not totally sure how to do.

**Comment by omahane** [comment]
@swirtSJW My change has been incorporated into 3.0.1 https://github.com/department-of-veterans-affairs/va.gov-cms/pull/11369

**Comment by swirtSJW** [comment]
https://github.com/department-of-veterans-affairs/va.gov-cms/pull/11369 was merged so this is closed.  Nice work.
